app/services/campaign_search_params_merger.rb

```diff
@@ -58,7 +58,9 @@ def extract_working_patterns(params)
 
   def extract_ect_status(params)
     ect_value = params.delete(:email_ECT)
-    params[:ect_statuses] = if ect_value.present? && ActiveModel::Type::Boolean.new.cast(ect_value)
```

Comment:
This was forcing a `ect_unsuitable` value into `params[:ect_statuses]` when the URL param wasn't provided.
We want only to override the config criteria if the URL param is explicitly given.